oshdb/src/main/java/org/heigit/ohsome/oshdb/OSHDBBoundingBox.java

```diff
@@ -24,7 +24,11 @@
   private final int maxLat;
 
   /**
-   * Creates an {@code OSHDBBoundingBox} instance from osm int coordinates.
+   * Creates an {@code OSHDBBoundingBox} instance from transposed coordinates.
+   * 
+   * This method is mainly for internal usage. Coordinates are stored as integer
```

Comment:
a `<p>` is needed here for correct javadoc formatting:

```suggestion
   * <p>This method is mainly for internal usage. Coordinates are stored as integers
```